lib/pure/json.nim

```diff
@@ -379,7 +386,13 @@ proc toJson(x: NimNode): NimNode {.compiletime.} =
     result = newNimNode(nnkTableConstr)
     for i in 0 ..< x.len:
       x[i].expectKind nnkExprColonExpr
-      result.add newTree(nnkExprColonExpr, x[i][0], toJson(x[i][1]))
+      var key = x[i][0]
```

Comment:
Why not simply do: ``var key = x[i][0].toStrLit``?

Reply:
Because that would force every key to be a string, no? But I want also to check whether some key is a variable that's defined by the user, so that variables can still be keys inside the `%*` macro.